stdcommands/roast/roastlist.go

```diff
@@ -8,7 +8,7 @@ func randomRoast() string {
 	return roasts[rand.Intn(len(roasts))]
 }
 
-//Roasts copied from roast-bot
+//Roasts copied from roast-bot and added to by WeepingHoney
```

Comment:
I don't think **you're** the original creator of these roasts, I have read them all before.

Reply:
no need for such self-inserts into the codebase, "added by" is already in repository history.

Reply:
idk what I'm doing, I just saw a line that wasn't going to be accurate anymore and updated it. Do you have a better suggestion as leaving it as it was would be inaccurate?

Reply:
Perhaps add where you sourced these roasts.